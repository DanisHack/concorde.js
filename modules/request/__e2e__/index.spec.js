describe('Request bundle', async () => {
  let page;

  beforeAll(async () => {
    page = await global.__BROWSER__.newPage();
    await page.goto(`${global.__CONFIG__.baseUrl}/request/__e2e__/index.html`);
  });

  it('should render an empty block, for now', async () => {
    await page.waitForSelector('#results');

    const results = await page.$eval('#results', (e) => e.innerHTML);
    expect(results).toMatchSnapshot();
  });
});
