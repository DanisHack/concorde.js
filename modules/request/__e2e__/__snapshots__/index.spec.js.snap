// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Request bundle should render an empty block, for now 1`] = `
"
  "
`;
